include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -65,7 +65,22 @@ class MGConstrainedDoFs : public Subscriptor
   template <int dim, int spacedim>
   void initialize(const DoFHandler<dim,spacedim> &dof,
                   const typename FunctionMap<dim>::type &function_map,
-                  const ComponentMask &component_mask = ComponentMask());
+                  const ComponentMask &component_mask = ComponentMask()) DEAL_II_DEPRECATED;
```

Comment:
can you add a "@deprecated Use initialize() followed by set_zero_boundary_dofs() instead."?